### ISSUE TITLE
[doctest] Segmentation fault (core dumped) make builds fail randomly

### ISSUE THREAD

**Issue by DelazJ** [qgis/QGIS-Documentation#5398]
Some PRs (https://travis-ci.org/github/qgis/QGIS-Documentation/pull_requests) are failing for the following reason (and this does not happen systematically)
```
Document: docs/user_manual/auth_system/auth_workflows
-----------------------------------------------------
Makefile:43: recipe for target 'doctest' failed
make: *** [doctest] Segmentation fault (core dumped)
docker.mk:4: recipe for target 'doctest' failed
make: *** [doctest] Error 2
The command "make -f docker.mk doctest" exited with 2.
cache.2
store build cache
```
Anyone knows what the issue could be?

**Comment by elpaso** [comment]
@havatv would you please point me to the failing PRs?

**Comment by DelazJ** [comment]
If I may, in place of @havatv ;)
Once I understood that restart building fixes the issue, I only used that tool instead of close/reopen the PR. The side effect is that the history of the failure is lost (afaics) in Travis and I'm unable to find any recent wrong report.
But digging into the history, I found https://travis-ci.org/github/qgis/QGIS-Documentation/builds/683118749 and https://travis-ci.org/github/qgis/QGIS-Documentation/builds/683106605 (both for the same #5395 that I had to close/reopen twice). Hope that helps.
